### DISCUSSION TITLE
Support number of retries option for Firestore transactions

### DISCUSSION THREAD

**Discussion by puf** [firebase/flutterfire#9155] [Feature request] [answered]
The native SDKs for iOS, Android, Web, Unity and C++ now all support an option to specify the maximum number of retries for a Firestore transaction, before it gives up. 

* https://firebase.google.com/support/release-notes/ios#9.2.0
* https://firebase.google.com/support/release-notes/android#2022-06-23
* 
* https://firebase.google.com/support/release-notes/unity#9.2.0
* https://firebase.google.com/support/release-notes/cpp-relnotes#9.2.0

It'd be great if the Flutter bindings for this SDK would also add this option.

**Comment by Lyokone** [comment, marked as answer]
I'll open a issue to track that: https://github.com/firebase/flutterfire/issues/9160

**Comment by Lyokone** [comment]
It's released :)

**Comment by puf** [comment]
Thanks for the quick follow-up Guillaume. 🙏
This feature is mentioned in the Firebase Release Notes video that is
coming out tomorrow, and it was really great to include Flutter in the list
of all platforms where it's available.
